### DISCUSSION TITLE
Dropdown filter search logic

### DISCUSSION THREAD

**Discussion by Venkatesh-VH** [primefaces/community#1193] [PrimeVue]
Hi,
Currently the dropdown filter is showing these results when I entered Ål, "Ålxxxx" and "Alxxx". Its considering Å === A. This is wrong. Is there a way to address this problem.?

Thanks

**Comment by FlipWarthog** [comment]
This might be a bug. cc @melloware , it looks like the same is true for PrimeReact. Both seem to be using `ObjectUtils.removeAccents` inside `FilterService` when filtering, which would convert something like "Å" to "A". Also seems like settings `filterLocale` to something like "no" (Norwegian) has no effect on this.

**Comment by melloware** [comment]
@FlipWarthog looks like i fixed this in PrimeFaces SelectOne years ago see this ticket: https://github.com/primefaces/primefaces/issues/4764

I think we can use `e.key.toLowerCase();` as the character to check for and not removeAccents.
